src/notebooklm/_chat.py

```
@@ -23,6 +24,34 @@
+class StreamChunk:
+    """A single chunk yielded by ask_stream().
+
+    Attributes:
+        text: Text content from this chunk (empty if this chunk has no text).
+        is_answer: True if this chunk is part of the main answer stream.
+        reference: A ChatReference if this chunk contains a citation.
+        conversation_id: The server-assigned conversation ID (may appear mid-stream).
+        done: True when all chunks have been delivered.
+    """
+
+    __slots__ = ("text", "is_answer", "reference", "conversation_id", "done")
```

Ruff (0.15.9)

[warning] `StreamChunk.__slots__` is not sorted

Apply a natural sort to `StreamChunk.__slots__`

(RUF023)